### DISCUSSION TITLE
What is Horizon Current Workload Wait

### DISCUSSION THREAD

**Discussion by joelatgrayv** [laravel/framework#52771] [Q&A]
What does this Wait value represent?

<img width="1223" alt="Screenshot 2024-09-12 at 1 06 43 PM" src="https://github.com/user-attachments/assets/774f217e-b73f-4caa-80bd-676f7f1842f6">

**Comment by henzeb** [comment]
I think it's the estimated time a new job needs to wait before it gets processed?

**Comment by joelatgrayv** [comment]
Thanks. If it has jobs to process, why would it wait 3 hours?

**Reply by henzeb** [reply]
Because you have so many jobs in queue that together take 3 hours before it can process any ones.
